official-http/node-fetch/index.js

```diff
@@ -0,0 +1,62 @@
+const fetch = require('node-fetch');
+const crypto = require('crypto');
+
+const apiKey = 'API_KEY';
+const apiSecret = 'API_SECRET';
+
+function makeRequest(verb, endpoint, data = {}) {
+  const apiRoot = '/api/v1/';
+
+  const expires = new Date().getTime() + (60 * 1000);  // 1 min in the future
+
+  let query = '', postBody = '';
+  if (verb === 'GET')
+    query = '?' + Object.entries(data).map(
```

Comment:
Please use the built-in `qs` module for this instead.

Reply:
Did you mean the built-in [querystring](https://nodejs.org/api/querystring.html) module? It doesn't support nested data. Would that be OK?

Reply:
`querystring` can't handle nested objects, such as `{filter: { symbol: 'XBTUSD' } }`, so I've used `qs` and made this into a library, [bitmex-plus](https://github.com/Bomper/bitmex-plus).